Modules/SubModule_Structural/xASL_wrp_VisualQC_Structural.m

```diff
@@ -105,7 +105,7 @@ function xASL_wrp_VisualQC_Structural(x)
 
 %% -----------------------------------------------------------------------------------
 %% 5) Final QCs
-xASL_qc_PrintOrientation(x.dir.SUBJECTDIR, x.P.Path_T1, x.dir.SUBJECTDIR, 'RigidRegT1');
+xASL_qc_PrintOrientation(x.P.Path_T1, x.dir.SUBJECTDIR, 'RigidRegT1');
```

Comment:
So this is probably the one where we have to be careful with #717, but looks good so far.

Reply:
Did you change the input arguments of `xASL_qc_PrintOrientation` in #717? That is confusing, I though #717 was about moving structures only... Are you sure here?

Reply:
OK